internal/handlers/pg/pgdb/database_metadata.go

```diff
@@ -219,7 +304,8 @@ func (m *metadata) remove(ctx context.Context) error {
 }
 
 // formatCollectionName returns collection name in form <shortened_name>_<name_hash>.
-// Changing this logic will break compatibility with existing databases.
+//
+// Deprecated: this function usage is allowed for collection metadata creation only.
```

Comment:
It will be done in #2239. Doing that also there will lead to more merge conflicts.

Reply:
No worries, let's merge #2239 first, and then I'll resolve the conflicts in the current PR. The code is mostly identical, so there shouldn't be many conflicts.

Reply:
ok, added "do not merge" label

Reply:
merged the changes from #2239.